app/src/main/java/chat/rocket/android/service/RealmBasedConnectivityManager.java

```diff
@@ -163,8 +165,8 @@ public void notifyConnecting(String hostname) {
       }
 
       return connectToServer(hostname)
-          //.doOnError(RCLog::e)
-          .retryWhen(RxHelper.exponentialBackoff(3, 500, TimeUnit.MILLISECONDS));
+          .doOnError(RCLog::e)
+          .retryWhen(RxHelper.exponentialBackoff(Integer.MAX_VALUE, 500, TimeUnit.MILLISECONDS));
```

Comment:
I agree it is. It's opened for discussion this max retry threshold. What I want to do on this one is set a high (not that high) one here and if we reach that max we defined then just display a RETRY button.